torch/testing/_internal/common_nn.py

```diff
@@ -2026,21 +2029,21 @@ def fractional_max_pool3d_test(test_case):
     ),
     dict(
         fullname='AvgPool2d_divisor',
-        constructor=lambda: nn.AvgPool2d((2, 2), divisor_override=1),
+        constructor=lambda: nn.AvgPool2d((2, 2), divisor_override=1),  # type: ignore[arg-type]
```

Comment:
This is a bug in the annotation of `AvgPool2d` (divisor_override should be `int` not `bool`) that should be fixed instead of ignoring it here.